### DISCUSSION TITLE
Home consumption

### DISCUSSION THREAD

**Discussion by mathschut** [slipx06/sunsynk-power-flow-card#426] [Sunsynk/Deye]
Hello, I have 3 inverters. An SMA Tripower with SMA Home Manager. A Deye with battery. I also have a small growatt. I have set everything up as far as possible, but unfortunately I can't see the house consumption. What am I doing wrong or what am I missing?

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/86230663/cd79f63e-453c-418c-943d-da8e78905e4c)

**Comment by slipx06** [comment]
HI. For the house consumption you need to provide a sensor for `essential_power:`

**Reply by mathschut** [reply]
Hi, thanks for your reply.  How can I calculate the value or where can I get the sensor because I have several inverters.

**Reply by slipx06** [reply]
Sorry I cant answer that because it depends on your specific setup and measurement points available. You may need to create a template sensor to sum the 3 inverters output on the load side.
